tokens/delivery/http/tokens_delivery.go

```diff
@@ -201,7 +223,45 @@ func (a *TokensHandler) GetPrices(c echo.Context) (err error) {
 	if err != nil {
 		return c.JSON(http.StatusInternalServerError, domain.ResponseError{Message: err.Error()})
 	}
-	return c.JSON(http.StatusOK, prices)
+
+	// If we are not in debug mode, we return the prices as is
+	if !debug {
+		result := make(GetPricesResponse, len(prices))
+		for baseDenom, quotePrices := range prices {
+			result[baseDenom] = make(map[string]osmomath.BigDec, len(quotePrices))
+			for quoteDenom, price := range quotePrices {
+				result[baseDenom][quoteDenom] = price.Price
+			}
+		}
+		return c.JSON(http.StatusOK, result)
+	}
+
+	// If we are in debug mode, we return the prices with routes
+	result := make(GetPricesDebugResponse, len(prices))
+	for baseDenom, quotePrices := range prices {
```

Comment:
Can we add a short README entry about the debug mode please?

Please Encapsulate:
- When we want to use this
-  How to activate this